compiler/lsp-tests/src/Main.hs

```diff
@@ -387,25 +390,43 @@ stressTests
 stressTests run _runScenarios = testGroup "Stress tests"
   [ testCase "Modify a file 2000 times" $ run $ do
         let fooValue :: Int -> T.Text
+            -- Even values should produce empty diagnostics
+            -- while odd values will produce a type error.
             fooValue i = T.pack (show (i `div` 2))
-                      <> if i `mod` 2 == 0 then "" else ".5"
+                      <> if even i then "" else ".5"
             fooContent i = T.unlines
                 [ "daml 1.2"
                 , "module Foo where"
                 , "foo : Int"
                 , "foo = " <> fooValue i
                 ]
+            expect i = when (odd i) $ do
```

Comment:
Could we please give this a type signature as a reminder what monad we're in.

Reply:
Done!